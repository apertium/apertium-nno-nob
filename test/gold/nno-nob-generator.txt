[z9L2UxItKoLA] Så hun ikke ordene? De småspiste nyplantet hygge.
[/z9L2UxItKoLA]
