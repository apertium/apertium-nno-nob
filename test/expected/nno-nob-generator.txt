[z9L2UxItKoLA#0] Så hun ikke ordene? De småspiste nyplantet hygge.
[/z9L2UxItKoLA]
